site/class-1-load/DesleyAalderink/DesleyAalderink/dinners.txt

```diff
@@ -0,0 +1,3 @@
+2017-9-23, meat

```

Comment:
Looks great, but why is there a `DesleyAalderink` in your `DesleyAalderink`? 🤔

Could you remove it?